proj/src/main/webapp/static/signup.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Welcome</title>
+    <!-- Bootstrap: Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <!-- Bootstrap: jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Bootstrap: Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="../style.css">
+    <!-- Font  -->
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container screen">
+      <div class="row fullscreen">
+        <!-- title  desktop:left    mobile:none -->
+        <div class="col-sm-8 info-graphic">
+          <div class="header">
+            <h1> camaraderie </h1>
+            <h4>a spirit of friendly good-fellowship</h4>
+          </div>
+          <div class="center-info">
+            <div id="chat-image-containter">
+              <img id="chat-image"src="./images/friendsChatting.jpg" alt="Friends chatting icon">
+            </div>
+            <div class="inblock-center-container">
+              <h3 class="black-bold-subtitle center-text"> meet friends <br> create communities</h3>
+              <button type="button" name="button" class="button">begin</button>
+            </div> 
+          </div>
+        </div>
+        <!-- sign up   desktop:left    mobile:top -->
+        <div class="col-sm-4 sign-up">
+          <div class="header">
+            <h1 id="sign-in-title"> sign up </h1>
+            <h4>welcome</h4>
+          </div> 
+          <div class="row justify-content-md-center">
+            <form>
+              <label for="fname">first name:</label><br>
+              <input type="text" id="fname" name="fname" class="input" required><br>
+              <label for="lname">last name:</label><br>
+              <input type="text" id="lname" name="lname" class="input" required><br>
+              <label for="email">email:</label><br>
+              <input type="text" id="email" name="email" class="input" required><br>
+              <label for="password">password:</label><br>
+              <input type="text" id="password" name="password" class="input" required><br>
+              <label for="tags">tags:</label><br>
+              <input type="text" id="tags" name="tags" class="input" required><br>
+
+              <button type="button" name="button" class="button">join</button>
+            </form>
+        </div>
+      </div>
+  </div>
+
+  <footer>
+    <div>Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
+  <footer>
+  </body>
+</html>
```

Comment:
Please merge with login/sign up system from index.html to register users to database. Check out authentication in scripts.js to see what element IDs are expected.

Reply:
Gotcha, I will make a new branch to work on merging things.